3-reveal/jobs/materialized-views/IRS/Zambia-2019/refresh_irs_reports.sql

```diff
@@ -1,4 +1,5 @@
 -- Suggestion is to refresh this every 15min
-SELECT refresh_mat_view('zambia_irs_structures', TRUE);
+SELECT refresh_mat_view('zambia_irs_structures', FALSE);
+SELECT refresh_mat_view('zambia_irs_structures_data', TRUE);
```

Comment:
After running overnight, it seems like splitting the view this way does avoid the tuple buildup - there's a longer-term trend of increasing baseline load though that worries me a bit - I'm not sure what's causing it - may be unrelated and/or just settling in.

![Screenshot from 2020-10-16 08-53-49](https://user-images.githubusercontent.com/52990406/96218333-32adfa00-0f8d-11eb-8f3b-1921e1b65a5f.png)